examples/asset_create_example.py

```diff
@@ -3,30 +3,60 @@
 from algosdk import account
 from algosdk.future import transaction
 
-private_key, address = account.generate_account() # creator
-_, freeze = account.generate_account() # account that can freeze other accounts for this asset
-_, manager = account.generate_account() # account able to update asset configuration
-_, clawback = account.generate_account() # account allowed to take this asset from any other account
-_, reserve = account.generate_account() # account that holds reserves for this asset
+private_key, address = account.generate_account()  # creator
```

Comment:
This seems unnecessarily complicated. I think if you moved the comments from the same line to the line before, the formatted code might be simpler?

Reply:
Thanks for catching this! Moving the comment does the trick, i.e. 
```
# account that can freeze other accounts for this asset
_, freeze = account.generate_account()
```